app/code/Magento/Catalog/Test/Mftf/Section/AdminEditProductAttributesSection.xml

```diff
@@ -9,14 +9,18 @@
 <sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdminEditProductAttributesSection">
-        <element name="AttributeName" type="text" selector="#name"/>
-        <element name="ChangeAttributeNameToggle" type="checkbox" selector="#toggle_name"/>
-        <element name="NameError" type="text" selector="#name-error"/>
-        <element name="AttributePrice" type="text" selector="#price"/>
-        <element name="ChangeAttributePriceToggle" type="checkbox" selector="#toggle_price"/>
-        <element name="PriceError" type="text" selector="#price-error"/>
-        <element name="AttributeDescription" type="text" selector="#description"/>
-        <element name="ChangeAttributeDescriptionToggle" type="checkbox" selector="#toggle_description"/>
+        <element name="AttributeName" type="text" selector="#attributes_name"/>
```

Comment:
Please change all selectors so that they do not violate https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html